### PR TITLE
Add functions for delete vpc at once

### PR DESCRIPTION
**Pull request by Baescott**
- Add functions for delete vpc at once

### CODE REVIEW

File: tests/test_s3_manager.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def test_get_bucket_encryption(sm, sample):
     bi = sm.get_bucket_encryption()
     assert bi
     print(bi['SSEAlgorithm'])
-    print(bi['KMSMasterKeyID'])
+    # print(bi['KMSMasterKeyID'])
```
Comment (lks21c) [inline tests/test_s3_manager.py:133]: remove comment plz

File: baram/ec2_manager.py [hunk 2]
```diff
@@ -402,13 +402,131 @@ def apply_imdsv2_only_mode(self,
...
+        :param destination_cidr_block:
+        :param route_table_id:
+        """
+        try:
+            self.cli.delete_route(DestinationCidrBlock=destination_cidr_block, RouteTableId=route_table_id)
+            self.logger.info(f'Route has deleted')
+        except:
+            print(traceback.format_exc())
+
+    def delete_internet_gateway(self, igw_id: str = '', vpc_id: str = ''):
```
Comment (lks21c) [inline baram/ec2_manager.py:420]: I recommend `delete_igw` instead.

File: baram/ec2_manager.py [hunk 3]
```diff
@@ -402,13 +402,131 @@ def apply_imdsv2_only_mode(self,
...
+        """
+        Delete internet gateway with its ID
+
+        :param igw_id:
+        """
+        all_igw_ids = [igw['InternetGatewayId'] for igw in self.cli.describe_internet_gateways()['InternetGateways']]
```
Comment (lks21c) [inline baram/ec2_manager.py:426]: what if `elf.cli.describe_internet_gateways()` doesn't have `['InternetGateways']`?

File: baram/ec2_manager.py [hunk 4]
```diff
@@ -402,13 +402,131 @@ def apply_imdsv2_only_mode(self,
...
+        try:
+            self.cli.detach_internet_gateway(InternetGatewayId=igw_id, VpcId=vpc_id)
+            self.cli.delete_internet_gateway(InternetGatewayId=igw_id)
+            is_deleted = igw_id in all_igw_ids
```
Comment (lks21c) [inline baram/ec2_manager.py:430]: do you mean `deleted candidate` as `is_deleted`?

File: baram/ec2_manager.py [hunk 5]
```diff
@@ -402,13 +402,131 @@ def apply_imdsv2_only_mode(self,
...
+            while is_deleted:
+                all_igw_ids = [igw['InternetGatewayId']
```
Comment (lks21c) [inline baram/ec2_manager.py:432]: why do you have to re-obtain all_igw_ids?

**Review by lks21c — CHANGES_REQUESTED** [review]

**Comment by Baescott** [comment]
This PR is closed because of the original issue has ceased.

**Pull request closed without merging**